generators/info/index.js

```diff
@@ -38,10 +38,10 @@ module.exports = ReportGenerator.extend({
 
         checkJHipster() {
             const done = this.async();
-            this.log('##### **JHipster Version(s)**');
+            console.log('##### **JHipster Version(s)**');
```

Comment:
I used `/* eslint-disable no-console */` at beginning of file and re-enabled it at the end to avoid cluttering the file with 19 line comments.
Please note that this works only with` /* */` comments not `//`